yo1k/sandbox/postgresql/pgexercises/readme.md

```diff
@@ -0,0 +1,11 @@
+Contains some solutions to programming exercises 
+from [pgexercises.com](https://pgexercises.com/gettingstarted.html)
+using [Psycopg](https://www.psycopg.org/docs/index.html) database adapter for the Python.
+
+It includes `clubdata.sql` script for creation a database.
+To create the database using a shell capable of running [Bash](https://www.gnu.org/software/bash/) 
+scripts,
+run the following from the directory with `clubdata.sql`:
+```shell
+$ psql -U postgres -f clubdata.sql -d postgres -x -q
+```
```

Comment:
I recommend always ending all files everywhere (this includes `clubdata.sql`) with an empty line. Also, consider placing an empty line before and after ``` blocks.

Reply:
Done.

By the way, file `clubdata.sql` has an empty line in the end. Perhaps you did not see it because of the auto-formatting of the github.